src/components/v2/Icon/icons/arrowShaft.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none"
```

Comment:
```suggestion
<svg width="16" height="16" viewBox="0 0 16 16" fill="currentColor"
```

This way we can use the `color` attribute directly. `fill` works too, but if you check the other icons so far we've been using `color` to colorize them.

Reply:
This is somethin we should probably do a find and replace for because many icons still use `fill="none"`